apps/docs/src/guide/contracts/proxy-contracts.md

```diff
@@ -2,9 +2,51 @@
 
 Automatic deployment of proxy contracts can be enabled in `Forc.toml`.
 
-Once that is in place, [fuels deploy](https://docs.fuel.network/docs/fuels-ts/fuels-cli/commands/#fuels-deploy) will take care of it.
+We recommend that you use [fuels deploy](https://docs.fuel.network/docs/fuels-ts/fuels-cli/commands/#fuels-deploy) to deploy and upgrade your contract using a proxy as it will take care of everything for you. However, if you want to deploy a proxy contract manually, you can follow the guide below.
 
-## Docs
+## Manually Deploying and Upgrading by Proxy
```

Comment:
I'd suggest separating the initial deployment and the upgrading into two sub headers.